ui/src/adapters/json.ts

```diff
@@ -33,14 +33,14 @@ export async function getJsonFromUrl({
 }: {
   signal?: AbortSignal;
   url: string;
-}): Promise<Response<Json>> {
+}): Promise<Response<Record<string, unknown>>> {
```

Comment:
Similar reason as before. I changed this because this function is used to retrieve the JSON Schema and the JSON LD Context and these are not allowed to be literal values such as `"foo"`, `3`, `true`, or `null`. These JSONs can only be objects. We can leave it as `Json` though. Reverted.